EpsilonLights/Tests/tests/LightsTest/LightsTest.c

```diff
@@ -0,0 +1,391 @@
+#include "unity.h"
+
+#include "Mockstm32f4xx_hal_gpio.h"
+#include "Mockstm32f4xx_hal_can.h"
+//#include "Mockstm32f4xx_hal_adc.h"
+#include "Mockcmsis_os.h"
+
+//#include "stm32f4xx_hal_can.h"
+#include "Lights.h"
+#include "LightsTest.h"
+#include <stdio.h>
+
+lightsInfo lightCharacteristics = {0};
+
+uint8_t lightsInputs;
+uint8_t auxBmsInputs[2];
+uint8_t batteryErrors[5];
+uint8_t driversInputs[4];
+SigLightsHandle sigLightsHandle;
+
+osMutexId* canHandleMutex;
+
+void runLightsTests()
+{
+    RUN_TEST(test_updateLights1_headLightsOffBrakesLightsOffHazardsOffBmsStrobeLightsOff);
```

Comment:
Generally, for naming you want to describe the situation that you want to test. Not straight up listing all the variables and their settings if you can avoid it. Also try not to have all caps in names unless it's suppose to be like that.